tests/PhoneValidatorTest.php

```diff
@@ -345,10 +346,11 @@ public function testValidatePhoneLenient()
         ]));
     }
 
+    /**
+     * @expectedException \Propaganistas\LaravelPhone\Exceptions\InvalidParameterException
+     */
     public function testValidatePhoneFaultyParameters()
     {
-        $this->setExpectedException('Propaganistas\LaravelPhone\Exceptions\InvalidParameterException');
```

Comment:
The reason I removed this was because it has been deprecated in phpunit 5.2. However I will add this back shortly.